policies/project-director-election-process.md

```diff
@@ -0,0 +1,93 @@
+    * This 10-day period is equivalent to the Change Round in the "Election Meeting" step; representatives may change their nominations in advance of the meeting, this does not require restarting the 10-day period.
+
+### Election Meeting
+
+The election meeting is comprised of all members of the Leadership Council who will actively participate in the consensus process as well as the facilitator who will facilitate that process.
+
+* Share nominations
+    * Write down nominations first[^1].
+    * Use a round format[^2] (e.g. [round format](https://www.sociocracyforall.org/on-rounds/))
+    * Nominations must include supporting reasoning 
+* Change Round
+    * During the change round, every team member will share their current preferred nominees and the reasons for changes from the previous preferred nominees if that applies.
+    * Note that the change round is driving around discussing and trying to select the best possible nominee(s) rather than discussing the full set of nominees. Council members are *not* de-selecting a nominee from consideration by not picking their name during the change round as their preferred choice.
+* Facilitator proposes a candidate:
+    * Based on gathered data, the facilitator suggests a candidate they believe will receive the group's consent.
+* Seek objections to the facilitator's proposal
+* Finalize the selection:
+    * Discuss and attempt to resolve any objections, if possible.
+    * Consent has been reached if the proposed choice has no remaining objections! :tada:
+    * If objections persist and the current proposal reaches an impasse, the facilitator should make a new proposal based on the discussion and feedback received.
+    * Continue seeking consensus and addressing objections until a satisfactory candidate is selected.
+        * Note that this process seeks to produce a candidate everyone is satisfied with and has no objections to. Avoid lengthy debates over preferences not related to objections or unmet requirements.
+* Document and announce the selection
+
+If for some reason the Council is unable to reach consensus on a group of
+candidates (for example, the Council finds blocking objections to all
+candidates), the Council will postpone the election process to give a chance to
+troubleshoot and find an approach that is more likely to be successful. We do
+not specify how this will happen, but the council would likely convene a
+subcommittee to work on the issue and report back to the council.
+
+### Conflicts of Interest
+
+The Council will follow the policies defined in the [Leadership Council
+RFC][rfc3392] for handling conflicts of interest. One potential source of
+conflicts of interest is if a Council member is nominated as a candidate for the
+Project Director role.
+
+[rfc3392]: https://rust-lang.github.io/rfcs/3392-leadership-council.html
+### FAQ
+
+#### Why is there no step where we require that people consent to be nominated?
```

Comment:
I'm confused about this section-- I thought [here](https://github.com/rust-lang/leadership-council/pull/12/files#diff-2556c5baab015a9c1c52a85c3b378a6a7f57bb50be65ff0111342c96bcc7c3ffR44) was the step where we require that people consent to be nominated:

> * Each team representative must share their team's nominations and the reasoning for their nominations first with the candidates being nominated, and then, for those candidates who consent to be nominated, with the council's selected facilitator.

Reply:
+1 to this being confusing. I believe the idea is that the person doesn't need to *initially* consent to a nomination. I believe we originally phrased this as they don't need to consent to being nominated *at all* and that consent comes at the point of accepting the role.